flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/TestHelpers.java

```diff
@@ -159,9 +159,9 @@ public static void assertRowData(
       return;
     }
 
-    Assert.assertTrue(
-        "expected Record and actual RowData should be both null or not null",
-        expectedRecord != null && actualRowData != null);
+    assertThat(expectedRecord != null && actualRowData != null)
```

Comment:
this should be `assertThat(expectedRecord).isNotNull()` and then assertThat(actualRowData).isNotNull()`.
We want to avoid using isTrue/isFalse as much as possible, because it's more difficult to debug when the assertion fails